### DISCUSSION TITLE
2FA via security key

### DISCUSSION THREAD

**Discussion by bnielsen1965** [npm/feedback#644] [Registry] [answered]
Are there plans to support security keys for 2FA authentication, i.e. the Yubico security key that provides U2F/FIDO2 authentication?

I've been using U2F/FIDO2 keys for various accounts for years and they are a very convenient alternative to devices that require batteries or can be damaged if dropped.

**Comment by MylesBorins** [comment, marked as answer]
Hey @bnielsen1965 we are in the process of implementing WebAuthn support which will include support for security keys

More details [in this blog](https://github.blog/2021-12-07-enrolling-npm-publishers-enhanced-login-verification-two-factor-authentication-enforcement/)
